### DISCUSSION TITLE
Welcome to Front-End-Checklist Discussions!

### DISCUSSION THREAD

**Discussion by thedaviddias** [thedaviddias/Front-End-Checklist#433] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ilyas-wise** [comment]
Let's get starting with other doors

**Reply by thedaviddias** [reply]
And windows too 😎

**Comment by Htou** [comment]
Do you know someone who has a back-end checklist?

**Reply by thedaviddias** [reply]
Nothing that I'm aware of yet.

**Comment by ryansamaeian** [comment]
Thanks soo much ! I love this github repo +1

**Reply by thedaviddias** [reply]
Thanks @Ryan-da-ghost!

**Reply by ryansamaeian** [reply]
Thank you @thedaviddias :) This saved my life!!!

**Comment by Koda-Black** [comment]
I love this repo so much but I'm yet to churn out anything meaningful and I get demotivated sometimes. I'm just not progressing at me expected/estimated rate.

**Reply by thedaviddias** [reply]
Let me know @Koda-Black if you need any help or motivation! We are all here to help each other! Sorry for the late reply!

**Reply by Koda-Black** [reply]
I'll be in touch @thedaviddias . Thanks for the offer man. God knows I need all the help I can get right now...lol. Really appreciate.

**Comment by Mehedi-MHS** [comment]
Thanks brother.This repo helped me a lot.But I have 2 questions.
1:Do I need to add sitemap in every page of a website?
2: I have added only .png format favicon(not .ico). But it is not working in Google search result.Now what should I do?

**Reply by thedaviddias** [reply]
Hey @Mehedi-MHS!
I'm glad it helped!
1: what do you mean by sitemap? sitemap.xml or sitemap page? If it's the sitemap.xml, you need to have the page you want bots like Google to crawl.
2: you also need to provide the icon in `ico` format I believe

**Reply by Mehedi-MHS** [reply]
Thanks again.I will also add .ico format favicon.
I have tried to mean sitemap.XML .
I have added sitemap in homepage of my website. Then submitted it to Google search console. But unfortunately googles crowling bot has indexed only homepage. Other pages are not showing in search results.

**Reply by ryansamaeian** [reply]
Because you need to crawl them all :)

**Comment by maxi3336** [comment]


**Reply by ryansamaeian** [reply]
good idea

**Comment by divonbriesen** [comment]
Just happened across this and love the idea... hoping to add some things but wondered if they're appropriate- can you eyeball this and let me know:? https://docs.google.com/document/d/1A5q7yJhMejSSi8Zoxn1YdGy55tLl_B-11WmmvOMo3ik/edit#heading=h.xra4qcp9wbr8
I'd start with the title element needing to incorporate both the site name and that page's name, else it's incomplete (and if a dynamic site, whatever is needed to regenerate that page).

**Reply by josephmoleka** [reply]
Cool

**Comment by ghost** [comment]
Thank you so much for the work !

**Comment by Doreen970** [comment]
Hey! Just bumped to this now. I cannot see the content yet.

**Reply by josephmoleka** [reply]
Lol

**Comment by rileyca** [comment]
Hello, just started on HTML. Good to be here!

**Reply by josephmoleka** [reply]
Ah bon???

**Reply by waltcode39** [reply]
what are you planning to do on html

**Comment by thisisnotdevin** [comment]
I have recently developed an interest in front-end development and I wanted to express my gratitude for this repository. Thank you so much, +1!

**Reply by waltcode39** [reply]
do you have a project you are working in

**Comment by Gerald-hash** [comment]
Hello, I'm Gerald. Just started learning front end web developing. Literally joined Github yesterday. Everything is overwhelming. Cant seem to find my way around stuff. Hopefully I will, and hopefully I wiil meet others who can help with projects on here. Happy coding folks!
